src/core/AudioEngine.cpp

```diff
@@ -86,16 +93,25 @@ AudioEngine::AudioEngine( bool renderOnly ) :
 	m_masterGain( 1.0f ),
 	m_audioDev( nullptr ),
 	m_oldAudioDev( nullptr ),
```

Comment:
Why these CTOR changes? These variables should be mostly unrelated?

I think a variable should only be initialized when it's initial value is really being read somewhere, so this just adds confusion: Users might think "this code somehow relies on m_fifo being NULL", but it does not.

Aside from this, our common opinion in LMMS seems to be to do CTOR initialization in the class (i.e. in the header file) when possible.